docs/project-model/project-json-reference.md

```diff
@@ -986,7 +854,9 @@ A short description of the project.
 For example:
 
     {
-        "summary": "This is my library."
+		"packOptions": {
+			"summary": "This is my library."
+		}
     }
```

Comment:
A mix of tabs and spaces is used here and in the following examples (e.g. 4 spaces before `{`, 2 tabs before `"summary"`).

Though it doesn't really matter, since it seems to render correctly in both GitHub and docfx.